pyproject.toml

```diff
@@ -18,6 +18,7 @@ dependencies = [
     "boto3",
     "dacite",
     "duckdb",
+    "clickhouse-connect==0.6.14",
```

Comment:
Can we leave this unpinned instead of pinning to a specific version? My general rule of thumbs is libraries should leave the dependencies as free as possible, and binaries can then pin them as needed.